hydromt_fiat/workflows/exposure_vector.py

```diff
@@ -803,13 +803,21 @@ def setup_max_potential_damage(
     def setup_ground_elevation(
         self,
         ground_elevation: Union[int, float, None, str, Path],
+        unit: str
     ) -> None:
         if ground_elevation:
             self.exposure_db["Ground Elevation"] = ground_elevation_from_dem(
                 ground_elevation=ground_elevation,
                 exposure_db=self.exposure_db,
                 exposure_geoms=self.get_full_gdf(self.exposure_db),
             )
+        # Unit conversion
+        if (unit == "meters" or unit == "m") and (self.unit == "feet" or self.unit == "ft"):
+            self.exposure_db["Ground Elevation"] = self.exposure_db["Ground Elevation"].apply(lambda x: x * 3.28084)
+            if "dep_subgrid" in Path(ground_elevation).stem:
+                self.logger.warning("SFINCS Ground Elevation is automatically converted from meters into feet")
```

Comment:
is this somethin that belongs to the front-end maybe? Because this is happening there. For example if someone used hydromt-FIAT without the GUI he would get this warning but the conversion would not take place except if they explicitly defined meters.